### PR TITLE
fix the node version issue and compile issue

### PR DESCRIPTION
**Pull request by jackyq2015**

### CODE REVIEW

File: contrib/docker/Dockerfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
-FROM alpine:3.2
-MAINTAINER Lars Gierth <larsg@systemli.org>
+FROM mhart/alpine-node:base-6
```
Comment (kpcyrd) [inline contrib/docker/Dockerfile]: Please do not use unofficial base images.

**Comment by kylerschin** [comment]
Looks fine... Good for merge? @cjdelisle 
Does anyone consider this project..... DEAD?

**Comment by kpcyrd** [comment]
Personally I'd prefer #1106 over this PR.

**Comment by cjdelisle** [comment]
I accepted #1106 so I am closing this. Please let me know if there are any other issues. Thanks !

**Pull request closed without merging**